pxtblocks/blocklyloader.ts

```diff
@@ -887,13 +887,17 @@ namespace pxt.blocks {
                 }
             }
 
+            if (currentlyVisible === handlerArgs.length) {
```

Comment:
`>==`, could it be that blockly source may have been corrupted or the API may have changed?